helm/examples/azure/hpcc-azurefile/templates/_helpers.tpl

```diff
@@ -94,11 +97,18 @@ spec:
     storage: {{ .plane.size }}
   accessModes:
     - {{ .plane.rwmany | default false | ternary "ReadWriteMany" "ReadWriteOnce" }}
-  azureFile:
-    secretName: {{ $common.secretName }}
-    secretNamespace: {{ $common.secretNamespace }}
-    shareName: {{ .plane.shareName }}
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: azurefile-csi
+  csi:
+    driver: file.csi.azure.com
     readOnly: false
+    volumeHandle: {{ .plane.volumeId | default (printf "100-%s" $shareName) }}  # make sure this volumeid is unique in the cluster
```

Comment:
question: is there any special significance to the "100-" prefix?

Reply:
https://kubernetes.io/docs/concepts/storage/volumes/

> volumeHandle: A string value that uniquely identifies the volume. This value must correspond to the value returned in the volume.id field of the CreateVolumeResponse by the CSI driver as defined in the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume). The value is passed as volume_id on all calls to the CSI volume driver when referencing the volume.

And seems it requires at least 1 or 2 digits in the string.

Reply:
This may be ok, as it's only an example and a default.
I haven't read these doc. links thoroughly but searching, I didn't see where it said re. 1 o 2 digits in string.